dev-docs/source/conf.py

```diff
@@ -24,11 +24,15 @@ def setup(app):
 extensions = [
     # we use pngmath over mathjax so that the the offline help isn't reliant on
     # anything external and we don't need to include the large mathjax package
-    'sphinx.ext.pngmath',
     'sphinx.ext.autodoc',
     'sphinx.ext.intersphinx'
 ]
 
+if sphinx_version > "1.8":
```

Comment:
You should use `LooseVersion` to compare, [example](https://github.com/mantidproject/mantid/blob/master/docs/runsphinx.py.in#L70), because `'1.10' > '1.8'` is `False`. Can you also change the other `sphinx_version` comparisons in both files.

Reply:
Good catch! Thanks, I will correct it tomorrow.